modules/ztp-upgrading-gitops-ztp.adoc

```diff
@@ -1,23 +1,26 @@
 // Module included in the following assemblies:
 //
-// *scalability_and_performance/ztp-deploying-disconnected.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-upgrading-gitops.adoc
 
 :_content-type: PROCEDURE
 [id="ztp-upgrading-gitops-ztp_{context}"]
-= Upgrading GitOps ZTP
+= Overview of the GitOps ZTP upgrade process
 
-You can upgrade the Gitops zero touch provisioning (ZTP) infrastructure independently from the underlying cluster, {rh-rhacm-first}, and {product-title} version running on the spoke clusters. This procedure guides you through the upgrade process to avoid impact on the spoke clusters. However, any changes to the content or settings of policies, including adding recommended content, results in changes that must be rolled out and reconciled to the spoke clusters.
+You can upgrade GitOps ZTP for a fully operational hub cluster running an earlier version of the GitOps ZTP infrastructure. The upgrade process avoids impact on managed cluster.
```

Comment:
You could provide the full name for the "ZTP" acronym before its first use in this module.